compiler/codegen/CodeGenRA.cpp

```diff
@@ -1606,12 +1606,6 @@ OMR::CodeGenerator::pickRegister(TR_RegisterCandidate     *rc,
                      callNode = node->getFirstChild();
                      }
 
-                  if (callNode &&
```

Comment:
Is it just me, or can you now edxtend the above block further down combining the block starting at 1607 with the block starting at 1609 (New file numbers)

Reply:
Yes.  The only way `callNode` can be set to something non-NULL is on L1606.  Please add this folding opportunity to this commit.